security/suricata/Makefile

```diff
@@ -76,7 +76,7 @@ LUA_CONFIGURE_ON=		--enable-lua \
 				--with-liblua-includes=${LUA_INCDIR} \
 				--with-liblua-libraries=${LUA_LIBDIR}
 
-LUAJIT_LIB_DEPENDS=		libluajit-5.1.so:lang/luajit-openresty
+LUAJIT_LIB_DEPENDS=		libluajit-5.1.so:lang/luajit
```

Comment:
Please keep this change to use luajit-openresty. All pfSense ports that need it are using this variant

Reply:
Reverted my change. This also came over from FreeBSD-ports upstream.